src/lib/post.ts

```diff
@@ -98,5 +100,26 @@ const getPostByTag = (tag: string, fields: Field[] = POST_FIELDS): Post[] => {
   return posts.filter((post) => post.tags.includes(tag));
 };
 
-export { getPostBySlug, getAllPosts, getPostByTag };
+const randomPostSlug = (excludedSlugs: string[] = []): string | null => {
```

Comment:
maybe we could keep the name `savedSlugs` too, same as the calling function, to be consistent? What do you think?